client/src/accounts/authentication/login.hook.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { describe, expect, it } from 'vitest';
+import { rest, server } from '~/mocks/server';
+import { act, renderHook, waitFor } from '~/test/utils';
+import { useLogin } from './login.hook';
+const ENDPOINT = '*/api/accounts/';
+
+interface Result {
+  isSuccess: boolean;
+  isError: boolean;
+  data: Record<string, unknown>;
+  mutate: (form: Record<string, unknown>) => void;
+  error: Record<string, unknown>;
+  current: Record<string, unknown>;
+}
```

Comment:
We should look to see if react-query provides better types for the likes of `mutate|error`, if not, we should create our own instead of defining them over and over, in separate tests. I'm sure you've already likely duplicated this elsewhere.